server/neptune/gateway/pr/signaler.go

```diff
@@ -168,6 +173,11 @@ func generatePRModeEnvSteps(cfg *valid.MergedProjectCfg, validateEnvs ValidateEn
 			EnvVarName:  "BASE_BRANCH_NAME",
 			EnvVarValue: validateEnvs.BaseBranchName,
 		},
+		{
+			StepName:    EnvStep,
+			EnvVarName:  "ATLANTIS_TERRAFORM_VERSION",
+			EnvVarValue: tfVersion,
+		},
```

Comment:
hmm i think for this it makes sense to just have a dedicated arg, since it's actually something that's essential for running the workflow.